platform_windows.go

```diff
@@ -30,19 +79,33 @@ func platformInvokeAfter(id uint64, after time.Duration) {
 
 // ----- Menu section -----
 
+var (
+	appBar *MenuBar
```

Comment:
I didn't think Windows had a concept of a global menu bar? Doesn't this need to be a per-window thing?

Reply:
It doesn't. This is part of the hack I'm currently doing with the `RootWindow`. I have some logic in place to clone the menu across all the windows, but I don't think it's quite working properly yet.